.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,34 @@
 <!-- Thank you for your pull request, please provide a brief summary of what this introduces (mandatory). Please point out any code that may be non-obvious to reviewers by using comments. -->
 
-<!-- Make sure that you've linted your files, written and run unit tests, and filled out or updated documentation (README) -->
-
 ## Summary
 
```

Comment:
My thought is `Summary` is a section and everything in it is included in the squash commit message. Just in the category though. For example the following description with a PR title of `fix(tooltip): Fix OverflowTooltip in IE11`:

```
## Summary

Fixes #1234

This change fixes overflow tooltips in IE11 on buttons with SVG icons.

## Category
Components

## Notes
Something important we want in the release notes.

---

## Checklist
...

## Additional notes

Stuff for PR review
```

Would produce the following commit message:
```
fix(tooltip): Fix OverflowTooltip in IE11

Fixes #1234

This change fixes overflow tooltips in IE11 on buttons with SVG icons.

[Category:Components]
Notes:
Something important we want in the release notes.
```

The `[Category:Components]` would be picked up in release notes so that this PR will be listed under `Components` heading.

The `Notes:` would be picked up in release notes as:
### Notes:
- fix(tooltip): Fix OverflowTooltip in IE11 (#1235)
    Something important we want in the release notes.